test/e2e/Functions.test.js

```diff
@@ -19,7 +19,7 @@ describe("end to end functions", () => {
     });
 
     test("function/arguments.brs", () => {
-        return execute(resourceFile("function", "arguments.brs"), outputStreams).then(() => {
+        return execute([ resourceFile("function", "arguments.brs") ], outputStreams).then(() => {
```

Comment:
This is a little cumbersome when there's only one argument.  I was wondering if a helper function like _executeFile()_ was justified, but it seems okay as is, too.